### DISCUSSION TITLE
Three H2 PCBs available out of Germany

### DISCUSSION THREAD

**Discussion by aaronknczn** [kachurovskiy/nanoels#101] [General]
I have ordered PCBs for the Nano ELS H2 and dont need all of them. If anybody is interested I would be willing to sell them for cheap and send them out.

**Comment by k-kristensen** [comment]
Hi
If you have a PCB leftover, I would be interested in buying.

**Reply by aaronknczn** [reply]
I still have some leftover. If you give me your Email Address or write to askon0902@gmail.com we can discuss the rest.

**Comment by aaronknczn** [comment]
Well I have one left but I wasnt planning on selling my last one.
Is there a reason you cant/dont want to buy from JLCPCB?
Because for 20€ plus shipping I would be willing to sell you my spare PCB,
however thats probably pretty much what youre going to pay with JLC for
five pieces.

Malte ***@***.***> schrieb am So., 21. Mai 2023, 22:03:
